app/models/customer_resource.rb

```diff
@@ -20,9 +20,37 @@ class CustomerResource < RmApiResource
 
   before_request do |name, request|
     if name == :find_by_package
-      request.get_params[:search] = ''
+      filters = request.get_params.delete(:filter) || {}
```

Comment:
so this is probably my bad for setting the precedent many 🌔 ago when i was first learning my way around Flexirest: but we really, really need to give these poor `before_request` blocks a rest. stack much more in here and it's liable to collapse, destroying nearby files in the process.  we could at the very least pull each section into a named method, but looking at how much repetition there is here i'm sure it could go way beyond that.  since that's not really in the scope of the ticket, i'm not gonna hold you accountable for a bad trend i started that has seemingly taken off in every model in the codebase.  but i think it's worth noting it, lest we forget to someday address this.